lms/envs/common.py

```diff
@@ -2174,6 +2174,13 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
```

Comment:
The throttle rates settings
```
      'user': '60/minute',
      'service_user': '120/minute',
```
these settings will only be used if API view has either `UserRateThrottle` or `ServiceUserThrottle` present in `throttle_classes` attribute.

since `ServiceUserThrottle` is added by this PR and will not be used. `UserRateThrottle` is used [here](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L70-L76) (`EnrollmentUserThrottle`  used [here](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L144), [here](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L255) and [here](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L457)) but its rate is hard coded in the class so the settings will be ignored.